### DISCUSSION TITLE
Vercel Web Analytics

### DISCUSSION THREAD

**Discussion by Jst-Tan** [vercel/community#4029] [Help]
### Summary

Hello! I wanted to ask for some assistance, I am planning to build a new website, which would be fully built using HTML, CSS and JavaScript, and could be hosted on Github Pages, Vercel.

I am considering of using Vercel Web Analytics for my new website, but it is come to my knowledge that Vercel don’t anonymise the IP, but just create a hash of the IP, which could be re-identify and it is consider as pseudoymisation, not anonymous. My new website is meant to not collect any personal data, since I want to make it entirely privacy-friendly and not subject to GDPR or any privacy laws around the world. 

### Steps to Reproduce

Anonymising the IP, like from an IP of  12.214.31.144 would be changed to 12.214.0.0 .

**Comment by amyegan** [comment]
Hey @Jst-Tan. You can find information about [Vercel Web Analytics](https://vercel.com/docs/analytics) privacy and compliance in the docs at https://vercel.com/docs/analytics/privacy-policy

> Vercel Web Analytics does not collect or store any information that would enable you to reconstruct an end user’s browsing session across different applications or websites and/or personally identify an end user.

> The lifespan of a visitor session is not stored permanently, it is automatically discarded after 24 hours.

**Reply by Jst-Tan** [reply]
Hi! I would like to ask, as per Vercel Web Analytics,

> Vercel Web Analytics allows you to track your website traffic and gather valuable insights without using any third-party cookies, instead end users are identified by a hash created from the incoming request.

Is a hash from the incoming request adding with user agent consider as personal data? I would like to ask on how to disable the hash from being created? And may I affirmative that no personal info is collected (including hashed IP, and not anonymous IP ), and no user could have a unique ID is created. 

I do not want to collect any data that could be subjected to any privacy laws through.

**Reply by Jst-Tan** [reply]
And may I ask if hash, is a hash create from the incoming request, or from the IP address?

**Reply by amyegan** [reply]
A hash created from the incoming request.
